src/foam/core/Validation.js

```diff
@@ -187,7 +187,7 @@ foam.CLASS({
             predicateFactory: function(e) {
               return e.GTE(self, self.min);
             },
-            errorString: `${self.label} must be at least ${self.min}.`
+            errorString: `Please enter a number greater than or equal to ${self.min}.`
```

Comment:
these should probably have the label in them so an error from the server is more actionable